src/communication/azext_communication/version.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.8.0b1'
+VERSION = '1.8.1'
```

Comment:
Please update the release history in this [file](https://github.com/Azure/azure-cli-extensions/blob/main/src/communication/HISTORY.rst)

Reply:
Thank you. I've updated that file.